app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ArticlesController < ApplicationController
+  
+  http_basic_authenticate_with name: "user", password: "secret", except: [:index, :show]
+
+  def index
+    @articles = Article.all
```

Comment:
Why don't you just return `Article.all` instead of assigning it to the `@articles` variable?

Reply:
I followed the guide, so i will change the return value with `Article.all`.

Reply:
@mauriciosabogal The values cannot be referenced by the view and @articles  now is a nil variable. So I have to return the @article variables